src/init.ts

```diff
@@ -184,7 +180,7 @@ async function generateTsConfig(options: Options): Promise<void> {
 export async function init(options: Options): Promise<boolean> {
   let packageJson;
   try {
-    packageJson = await readJson('./package.json', noop, true /*strict*/);
+    packageJson = await readJson('./package.json');
```

Comment:
Would `require('./package.json')` work?

Reply:
I don't think `require` will work since it loads a file relative to `build/src/init.js`.

Reply:
How about `require(path.join(process.cwd(), 'package.json'))`?

Reply:
That would work, but in my opinion we should prefer explicitly reading a file over using a function that is meant to load modules. In this case, we are reading a JSON file, modifying its contents, and then overwriting it. Even though it's a totally valid operation to `require` a JSON-file, I think its use should be restricted to files that are treated as data-only modules (such as a constants file).

Reply:
It is not quite accurate to say that `require` is meant to load modules. Node core fully supports load JSON files.

If we should have a `readJSON` function, we need to be able to articulate how `require` is inadequate. You have stated preferences, but not a rationale.

Reply:
`require` is by no means inadequate, but I would still argue that it is meant primarily to load modules. My rationale is that there's a distinction between "module" and "data" JSON files -- "module" JSON files being portable/shorthand form for JS modules that only export static values (and therefore part of the application's source code), and "data" JSON files being among the application's I/O. That they happen to have the same file extension and data format should be irrelevant. I would consider reading and writing the user's `package.json` file to be the latter.

Reply:
I don't agree, but this was a nit, and if you feel strongly about this, I am not going to block.